src/transformers/models/squeezebert/tokenization_squeezebert_fast.py

```diff
@@ -60,18 +65,153 @@
 }
 
 
-class SqueezeBertTokenizerFast(BertTokenizerFast):
+class SqueezeBertTokenizerFast(PreTrainedTokenizerFast):
```

Comment:
Same here!

Reply:
Do you mean to write something like `# Copied from transformers.models.bert.tokenization_bert.BertTokenizerFast with Bert->SqueezeBert,BERT->SqueezeBERT` above this?

Reply:
Yes!